components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2886,6 +2751,78 @@ subroutine get_rain_dsd2(qr,nr,mu_r,rdumii,dumii,lamr,mu_r_table,cdistr,logn0r,r
 
   end subroutine get_rain_dsd2
 
+  !===========================================================================================
+  subroutine get_ice_dsd2(qr,nr,mu_r,rdumii,dumii,lamr,mu_r_table,cdistr,logn0r,rcldm)
+
```

Comment:
wouldn't the input variables for ice_dsd end in i rather than r? This shouldn't change the solution, but would be more logically consistent.

Reply:
ahhh, yes, this can just be removed entirely.  This is legacy from when I was experimenting with how to calculate epsi in a way consistent with P3.  This is no longer used.